packages/helix/brioche.lock

```diff
@@ -0,0 +1,12 @@
+{
+  "dependencies": {
```

Comment:
The dependencies here should be empty: `"dependencies": {}`. Since we are in a Brioche workspace [here](https://github.com/brioche-dev/brioche-packages/blob/main/brioche_workspace.toml), we can directly make reference to other packages. I think, if you re-run a Brioche command to build the package or run the test method, Brioche will automatically update the lock file and suppress the dependencies.

Reply:
Ooh, interesting! @jokeyrhyme did you originally have `helix` in a separate dir then moved it into `brioche-pacakages`? If so then I see how the lockfile ended up like this, otherwise I'm not sure how the lockfile would've ended up like this...

Reply:
yeah, i developed these elsewhere including running `brioche install` on them, and then moved them to their current locations for the PR

you can relax, haha :)